ev3dev/display.py

```diff
@@ -228,18 +238,6 @@ def draw(self):
         """
         return self._draw
 
-    @property
-    def image(self):
-        """
-        Returns a handle to PIL.Image class that is backing the screen. This can
-        be accessed for blitting images to the screen.
-
-        Example::
-
-            screen.image.paste(picture, (0, 0))
-        """
-        return self._img
```

Comment:
I think we should do what we can to keep this accessor in some form, as it's what all existing display code is based on.

Reply:
I'm trying to remember why I removed this and can't come up with a good reason....will put this back and rename the other `def image` to `def image_filename`